.github/workflows/ci.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+env:
+  NODEJS_VERSION: v18.18.2
 jobs:
```

Comment:
@outSH I've seen this in some other diff of another PR - maybe there's a dependency in this PR to that PR so that the other one goes in first?